projects/managed-hubs/support.md

```diff
@@ -1,118 +1,214 @@
+6. **Resolve the request**. Our team should work together to prioritize and resolve this request according to its impact.
+7.  **Confirm resolution**. Once we have resolved a support request, send a message to the Community Representative to confirm that we believe it is resolved. In FreshDesk, mark the incident as {guilabel}`Resolved`.
+8.  **Close the request**. If the Community Representative confirms that their request has been fulfilled, or if we have not gotten a response after 48 working hours, consider this request closed. In FreshDesk, mark the incident as {guilabel}`Closed`.
 
-- Ask questions that will help us understand whether an issue is "critical". For example, "Does this effect many users on the hub, or just a subset?", "Does this problem always happen, or intermittently?", "Is the hub accessible at all or is it totally down?".
-- Events should be prioritized (or in general, moments with a large influx of users)
-- In general, "how much does this incident impact the users of a given hub?" is a good guideline for deciding what to prioritize
+(support:prioritize-requests)=
+## How to prioritize non-incident support requests
 
+For support requests that are not related to {term}`Incident`s, we have more flexibility in when to do the work.
+We cannot sustainably prioritize all support requests over our other work, so here is a rough prioritization order to follow:
 
-## Asking for help from others
+1. Requests that come with dedicated funding.
+1. Requests that impact many communities.
+1. Requests that align with our pre-existing goals.
+1. Requests that align with an upstream open source project's goals.
+1. Requests from communities that haven't used much {term}`Support Budget`.
+1. Requests that have a high-impact on a single community.
+1. Our other work items
+1. Requests from communities that have no more {term}`Support Budget`.
 
-The support team is primarily a **communicator** and a **router** - they are not necessarily the ones who carry out the changes needed to resolve a support issue.
-Instead, the support team is empowered to ask engineering team members to pick up backlog items that are related to support requests.
+### Support request key terms
 
-When a support request is made that requires an action from a 2i2c engineer, a Support Steward should describe this change in a GitHub issue, and add it to the [Sprint Board](coordination:sprint-board).
-Think about an engineering team member that likely has the skills and capacity needed, and ask them if they are willing to take on resolving this issue.
-Try not to ask the same person for support help many times in a row - we should spread the work needed to address support issues across the team.
+The following are other important terms and ideas in the support process.
 
-## Who can make support requests?
+```{glossary}
+Support Budget
```

Comment:
You mentioned above some hours are allocated as Support Budgets, what is the amount we are talking about?

Reply:
I know you added a note above about it... just wondering if you have a number in your head.

Reply:
I am not sure haha, but here is some thinking about a rationale for arriving at a number.

- Assume 173 working hours a month per engineer
- Assume any given engineer should spend no more than 20% of their time (one day a week) on average dealing with support requests
- That amounts to 34 hours a month on support requests.

Then we can draw this number back to our question "how many hubs should be able to sustain a single engineer?". That ranges between 20 and 8 (depending on edu/research, and dedicated cluster). So as an example, if we use those as limits, we could say that those various hub types correspond to 34/20 ~= 1 hours, or 34 / 8 ~= 4 hours of support each month.

I think that more generally, it'd be better to just more explicitly tie the idea of "more support budget" to "higher monthly cost", but maybe this is a starting point for conversation? Don't think too much about the specific numbers here, I'm wondering if this general approach to arriving at a number makes sense. (also welcome to feedback about specific numbers as well)

Reply:
I think the rationale makes sense as a first iteration. When we have measurements and real data about the time we spend on support, we will have a better understanding and we may tweak/adjust the values (or even the rationale).